src/tr1/game/overlay.c

```diff
@@ -415,7 +417,7 @@ static void M_DrawPickup3D(DISPLAY_PICKUP *pu)
         }
 
         Matrix_TranslateRel(bone->pos.x, bone->pos.y, bone->pos.z);
-        Matrix_RotXYZ16(&frame->mesh_rots[i]);
+        Matrix_RotXYZ16(mesh_rots[i]);
```

Comment:
Can we use ``frame->mesh_rots` directly?

Reply:
Sure thing, done. I've left functions like `Lara_Draw` with the variables, as these get re-assigned at various stages for arms and what-not. LMK if that looks OK.

Reply:
Yes these are fine, I meant only trivial cases. Thanks.